docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -529,7 +529,10 @@ You cannot enable the following plugins, because they potentially harm the stabi
 - Kubernetes Filter
 - Rewrite_Tag Filter
 
-In addition, the prefix `__k8s__` is reserved by the LogPipeline and should not be used in any Filter definition.
+### Reserved log attributes
+The log attribute with name `kubernetes` is a special attribute being enriched by the `kubernetes` filter. When using that attribute as part of your structured log payload, the metadata enriched by the filter gets overwritten by the payload data. Filters which are relying on that metadata might not work anymore as expected.
+
+Furthermore, the prefix `__k8s__` is used for internals. When using the attribute prefix in your log data, the data might get overwritten.
```

Comment:
"internal" what? Internal use? Internal users?

```suggestion
Furthermore, the prefix `__k8s__` is used for internals. When you use the attribute prefix in your log data, the data might be overwritten.
```

no "get" passive in our docs.